src/afl-fuzz-queue.c

```diff
@@ -762,6 +762,9 @@ void add_to_queue(afl_state_t *afl, u8 *fname, u32 len, u8 passed_det) {
 
   q->skipdet_e = (struct skipdet_entry *)ck_alloc(sizeof(struct skipdet_entry));
 
+  q->fs_meta = NULL;
+  q->fs_status = 0;
+
```

Comment:
it is freshly allocated so all 0/null, so you can remove this. (and while doing this please remove the already present
```
  q->trace_mini = NULL;
  q->testcase_buf = NULL;
  ```
  as well :-) thanks